torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -382,8 +391,11 @@ static std::
     }
   }
   for (auto& ncclComm : allNCCLComms) {
-    std::string ncclUniqueIDStr = buildNcclUniqueIdStr(ncclComm->getNcclId());
-    ncclDumpMap[ncclUniqueIDStr] = ncclComm->ncclCommDump();
+    std::stringstream ss;
+    ss << std::hex << ncclComm->getNcclUniqueHash();
+    std::string ncclUniqueHashStr = ss.str();
```

Comment:
```suggestion
    std::string ncclUniqueHashStr = fmt::format("{:x}", ncclComm->getNcclUniqueHash());
```
More efficient and readable to use libfmt here?